interfaces/tempo_cluster/v0/schema.py

```diff
@@ -0,0 +1,78 @@
+"""This file defines the schemas for the provider and requirer sides of this relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+from enum import Enum
+from typing import Optional, Dict, Any, Literal
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel
+
+ReceiverProtocol = Literal[
+    "zipkin",
+    "otlp_grpc",
+    "otlp_http",
```

Comment:
Should `jaeger_thrift_http` be here? https://www.jaegertracing.io/docs/1.58/apis/#thrift-over-http-stable

I think it might be a bit premature to drop it yet.

Reply:
we decided to drop it from tempo and potentially add it back later if needed. Question is, should we say in the spec that the provider is supposed to offer any subset of all possible protocols, or all of them? 
If the former, we can add jaeger here (but not in tempo); if the latter, we'd need to add it to both or neither.

Reply:
As discussed offline, I'm leaning towards allowing the provider to offer a subset of protocols. That could be beneficial once tempo stops supporting any of the ingestion protocols or adds a new one.

This is a more general topic for `tracing` interface as well, though :)

Reply:
conclusion of 1:1 chat: The list of protocols is subject to growing and changing, it depends on what the provider (the version you're deploying) supports. That is also specified in the tracing protocol in this repo.